wd_helper.au3

```diff
@@ -2098,9 +2098,9 @@ Func __WD_JsonElement($sElement)
 EndFunc   ;==>__WD_JsonElement
 
 ; #FUNCTION# ====================================================================================================================
-; Name ..........: __WD_JsonAction
+; Name ..........: _WD_JsonAction
```

Comment:
@Danp2
Why did you change __WD_JsonAction >> _WD_JsonAction
I was thinking about this function as an internal .

Reply:
> Why did you change __WD_JsonAction >> _WD_JsonAction I was thinking about this function as an internal .

@mlipok I did this because I specifically stated that this function would be able to be called by an external routine, and it can be used by the script writer to more easily build webdriver actions.

Reply:
Ok